.codex/mcp.json

```diff
@@ -1,3 +1,10 @@
 {
-  "servers": []
-}
\ No newline at end of file
+  "mcpServers": {
```

Comment:
I applaud the default inclusion of playwright as it's awesome.  But I do wonder if others would love being forced to include it.  You might just wanna change the name of the key to mcpServers here.